src/azure-cli-core/azure/cli/core/auth/util.py

```diff
@@ -41,11 +41,21 @@ def aad_error_handler(error, **kwargs):
     error_codes = error.get('error_codes')
 
     # Build recommendation message
+    recommendation = None
     if error_codes and 7000215 in error_codes:
         recommendation = PASSWORD_CERTIFICATE_WARNING
     else:
-        login_command = _generate_login_command(**kwargs)
-        recommendation = "Interactive authentication is needed. Please run:\n{}".format(login_command)
+        login_command = _generate_login_command(claims_challenge=claims_challenge, **kwargs)
```

Comment:
Is it worth replacing `kwargs` with `tenant` and `scopes`, given the limited number of arguments?